packages/nextjs/utils/toast.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
```

Comment:
Created it inside `packages/nextjs/utils`, please let me know if we should move it to `packages/nextjs/components`

also please let me if the current structure and how the toast are called is good or not, happy to revamp and give it an another try 🙌